core/src/main/java/se2/ticktackbumm/core/models/BombImpl/Bomb.java

```diff
@@ -47,13 +51,15 @@ public Bomb(){
 
         stage = new Stage();
 
+        networkClient = tickTackBummGame.getNetworkClient();
+
         bombTexture = assetManager.get("bomb/bomb.png",Texture.class);
         bombImage = new Image(bombTexture);
 
         bombState = BombState.NORMAL;
 
         radnomExplosion = new SecureRandom();
-        explodeTime = (float)radnomExplosion.nextInt(10)+1;
+        explodeTime = radnomExplosion.nextInt(10)+1;
```

Comment:
Typo in `radnomExplosion`.
Die Zeit sollte hier eher vom Server kommen, als von der Bombe gesetzt zu werden.

Reply:
Ja stimmt, macht Sinn. 👍  Aber ich bin mir nicht sicher wie, weil ich  nicht auf Daten aus package server  zugreiffen kann und in Bomb class aufrufen  :/